torch/_dynamo/variables/dicts.py

```diff
@@ -153,10 +153,14 @@ def call_method(
             and args
             and ConstDictVariable.is_valid_key(args[0])
             and ConstDictVariable.get_key(args[0]) not in self.items
-            and len(args) == 2
+            and len(args) in (1, 2)
         ):
-            # missing item, return the default value
-            return args[1].add_options(options)
+            if len(args) == 2:
+                # missing item, return the supplied default value
+                return args[1].add_options(options)
+            else:
+                # missing item, no default value, return None
+                return ConstantVariable(None)
```

Comment:
This is correct if ```name == "get"``` but it throws ```KeyError``` for missing item if ```name == "pop"`, can you update it to reflect the difference? Thanks!

Reply:
How do I raise an error properly? Something like this just crashes Dynamo itself:
```
            if len(args) == 2:
                # missing item, return the default value
                return args[1].add_options(options)
            elif len(args) == 1:
                if name == "get":
                    return ConstantVariable(None)
                else:
                    raise KeyError(args[0])
```

Reply:
You can just call ```unimplemented("pop missing key {args[0]}")``` to fallback to eager.

Reply:
fixed